docs/src/contribute/design/next-tedge-api/case-studies/rentable-equipment.md

```diff
@@ -0,0 +1,272 @@
+
+Focusing on a single pump, the data for the pump needs to be easily tracked to monitor the operating hours to plan the pump's servicing. The company manufacturing the pump knows when the pump but previously they were not able to monitor the usage of the pump whilst it was rented out. Due to the lack of information, an estimated usage per rental period was calculated, however this would often be wrong as the each customer's usage patterns would be wildly different. As a result, the pumps were often failing due to be over-worked.
+
+### Overview
+
+Since the equipment (in this case a pump) is being rented out to different customers, ideally the id of the pump should remain the same across different installations.
+
+Let's model the pump as a child device in thin-edge, but the pump which is connecting to thin-edge is not very interesting, so thin-edge should be treated as a transparent gateway where there gateway should not influence the identity of the pump itself. This allows the pump to be moved around, and it could even be used against different thin-edge gateways during its lifetime.
+
+Each customer can rent any number of pumps at one time, and the rental period for each pump may be different (depending on the customer's need). In order to monitor the pumps, the pumps connect to a gateway (in this case thin-edge), and thin-edge handles the Cloud IoT aspects (connectivity, device management, monitoring etc.).
+
+Customer1 rents out two pumps; **pump1** and **pump2**. However the customer does not need both pumps for the entire time, so **pump2** will be returned after 2 weeks, whereas **pump1** has been rented out for a few months.
+
+```mermaid
+graph
+  cloud --> gateway1
+  subgraph Customer1
+    gateway1 --> pump1
+    gateway1 --> pump2
+  end
+```
+
+**Customer1** then returns **pump2** after the 2 weeks, then the same pump is rented out to a different customer, **Customer2**. The diagram shows the new overview tracking where each pump is located.
+
+```mermaid
+graph
+  cloud --> gateway1
+  cloud --> gateway2
+  
+  subgraph Customer2
+    gateway2 --> pump2
+  end
+  subgraph Customer1
+    gateway1 --> pump1
+  end
+```
+
+### Topic structure
+
+In this case, thin-edge should act more like a transparent gateway where each device is fully capable to describe itself.
+
+Each pump will be modelled as a child device of the thin-edge device, as thin-edge still needs to be used to facilitate the communication, however each child device should be uniquely identifiable via its own information.
+
+Therefore, the topic structure to facilitate this will be give different semantic meanings to the 4-part topic structure offered by thin-edge.
+
+```text
+te/[ManufacturerId]/[SerialNumber]/[Application]/[Instance]/[Channel]
+```
```

Comment:
Don't we need the `customer` info for `pump` mapping? For example, if a customer1 rents n-pumps, then if for some use cases, these only have to be filtered, how this can be done?

Reply:
In this scenario, the company renting out the customer tracks the name of the customer which currently has the pump purely in the cloud (e.g. by adding a fragment to the digital twin object of the pump). But the main point is that the topic name should be agnostic to the customer, because the customer renting the equipment changes over time.